"make stats" fails because it cannot find scc

This is probably something simple that's missing from the developer setup step. But when I run `make stats`, I get the following error:

```
xargs: scc: No such file or directory
make: *** [Makefile:80: stats] Error 127
```

--- a/documentation/development/setup.md
+++ b/documentation/development/setup.md
@@ -11,6 +11,7 @@ websites for help getting them installed.
   - Mac and Linux users should be okay out of the box
   - Windows users should install
     [Make for Windows](http://gnuwin32.sourceforge.net/packages/make.htm)
+- [scc](https://github.com/boyter/scc)
 
 Fork Git Town and clone your fork into a directory outside your GOPATH. Git Town
 uses Go modules and doesn't work properly inside the GOPATH.

